### ISSUE TITLE
Add image to circles

### ISSUE THREAD

**Issue by dehlen** [romaHerman/timeLineiOS#6]
Is it possible to add an image to the cirlces by customizing the specific CALayer ? 
I tried this, but without luck:
```
in: - (void)addProgressBasedOnLabels:(NSArray *)labels currentStatus:(int)currentStatus {

circleLayer.contents = (__bridge id)[UIImage imageNamed:@"myImage"].CGImage;
```

**Comment by romaHerman** [comment]
@dehlen , I will think how to add this feature 

Could you add some details, so I can understand better: you want an image inside circles?

**Comment by dehlen** [comment]
Yes right. I added the possibility to add some ovals to your library. Here you can find an example:
![bild1](https://cloud.githubusercontent.com/assets/680180/5798025/6c5d2bf8-9fc3-11e4-9b0e-482724c7de68.png)

Now i want to add the possibility to add images onto that CALayer. I think this image makes clear what i am trying to achieve:
![bild2](https://cloud.githubusercontent.com/assets/680180/5798026/6f79bf86-9fc3-11e4-9633-e795fd563f9e.png)

**Comment by romaHerman** [comment]
Yes I understand 
Thank you 
I will try to add functionality soon

**Comment by dehlen** [comment]
Great to hear that. Keep up the good work :)

**Comment by dehlen** [comment]
Something new regarding this issue ?
